runtime/compiler/ilgen/Walker.cpp

```diff
@@ -7012,6 +7186,89 @@ TR_J9ByteCodeIlGenerator::storeFlattenableInstanceWithHelper(int32_t cpIndex)
    genTreeTop(helperCallNode);
    }
 
+void
+TR_J9ByteCodeIlGenerator::storeFlattenableInstance(int32_t cpIndex)
+   {
+   /* An example on what the tree with flattened fields would look like
+    *
+    * class AssortedReftWithLongAlignment {
+    *    flattened Point2D point;
+    *    flattened Line2D line;
+    *    flattened ValueObject o;
+    *    flattened ValueLong l;
+    *    flattened ValueDouble d;
+    *    flattened ValueInt i;
+    *    flattened Triangle2D tri;
```

Comment:
I don't think all these fields are relevant for this example.  If they're not, I suggest you remove them to avoid confusion.